packages/forms/src/UIForm/utils/properties.js

```diff
@@ -31,3 +31,15 @@ export function omit(properties, key) {
 		});
 	return result;
 }
+
+/**
+ * Convert a string value to the wanted type
+ * @param type The string type
+ * @param value The value to convert
+ */
+export function convertValue(type, value) {
```

Comment:
I put it here because it convert it to be stored in the properties. It's like a properties cleaning/preparation.
If you insist on putting it out after my explanation, I can do it.